packages/material-ui-system/src/index.js

```diff
@@ -2,7 +2,7 @@ export { css, keyframes, GlobalStyles, StyledEngineProvider } from '@material-ui
 export { default as borders } from './borders';
 export * from './borders';
 export { default as breakpoints } from './breakpoints';
-export { handleBreakpoints, mergeBreakpointsInOrder } from './breakpoints';
+export { handleBreakpoints, mergeBreakpointsInOrder, resolveBreakpointValues } from './breakpoints';
```

Comment:
Could we export it as unstable_ so we can delete it later on if needed?
```suggestion
export { handleBreakpoints, mergeBreakpointsInOrder, resolveBreakpointValues as unstable_resolveBreakpointValues } from './breakpoints';
```